tests/unittests/classification/test_precision_recall.py

```diff
@@ -395,7 +395,7 @@ def test_multiclass_precision_recall_half_gpu(self, inputs, module, functional,
     ("k", "preds", "target", "average", "expected_prec", "expected_recall"),
     [
         (1, _mc_k_preds, _mc_k_target, "micro", tensor(2 / 3), tensor(2 / 3)),
-        (2, _mc_k_preds, _mc_k_target, "micro", tensor(1 / 2), tensor(1.0)),
+        (2, _mc_k_preds, _mc_k_target, "micro", tensor(1.0), tensor(1.0)),
```

Comment:
let's keep the original case too?

Reply:
Sorry, I didn’t understand. Could you please clarify?

Reply:
```suggestion
        (2, _mc_k_preds, _mc_k_target, "micro", tensor(1 / 2), tensor(1.0)),
        (2, _mc_k_preds, _mc_k_target, "micro", tensor(1.0), tensor(1.0)),
```